spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  describe 'Index page' do
+    let!(:user) do
+      User.create(name: 'Matias',
+                  photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1974&q=80',
+                  bio: 'Bio for Matias')
+    end
+
```

Comment:
* Well done written all integration tests but just one little issue about solving N+1 problems in your code I can see on the screenshot you attached to your PR it was solved but on your code it isn't the purpose of this is to adhere to best practice and ensure that your code is fast and efficient, kindly solve the N+1 issue in your post controller index action 
![image](https://github.com/Mar12358/blog-app/assets/52497246/510a6ca0-9ac9-4d0a-ba5c-e2d07e31980e)